Sources/Application/AppEnvironment.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+public final class AppEnvironment {
+    enum Key: String {
+        case authorizeEndPoint = "Authorize Endpoint"
+        case tokenEndpoint = "Token Endpoint"
+        case redirectURL = "Redirect URL"
+        case clientId = "Client ID"
```

Comment:
`clientID` would be more consistent with our other repos 😄

Reply:
what casing is that??